packages/@aws-cdk/aws-lambda-nodejs/lib/bundling.ts

```diff
@@ -122,21 +134,38 @@ export class Bundling implements cdk.BundlingOptions {
 
   private createBundlingCommand(options: BundlingCommandOptions): string {
     const pathJoin = osPathJoin(options.osPlatform);
+    const npx = options.osPlatform === 'win32' ? 'npx.cmd' : 'npx';
+
+    let compileCommand = '';
+    let relativeJsEntryPath;
```

Comment:
```suggestion
    let relativeEntryPath = this.relativeEntryPath;
```
when precompiling we overwrite this value and then we can always pass `pathJoin(options.inputDir, relativeEntryPath)` to `--bundle`.